patientsearch/models/sync.py

```diff
@@ -17,9 +17,9 @@ def add_identifier_to_resource_type(bundle, resource_type, identifier):
         return result
 
     for resource in result["entry"]:
```

Comment:
would it not be more clear, to set `resource` to the nested `resource`?

i.e.
```
for entry in result["entry"]:
    resource = entry["resource"]
    ...
```

Reply:
thanks- using `entry` for the loop var is a little clearer; done!